src/main/java/com/weirddev/testme/intellij/ui/classconfigdialog/CreateTestMeDialog.java

```diff
@@ -0,0 +1,197 @@
+package com.weirddev.testme.intellij.ui.classconfigdialog;
+
+import java.util.*;
+import java.util.List;
+
+import javax.swing.*;
+
+import com.intellij.psi.*;
+import com.intellij.psi.util.InheritanceUtil;
+import com.weirddev.testme.intellij.common.utils.PsiMethodUtils;
+import com.weirddev.testme.intellij.template.TemplateRegistry;
+import com.weirddev.testme.intellij.template.context.*;
+import org.jetbrains.annotations.NotNull;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.DialogWrapper;
+import com.intellij.openapi.util.NlsContexts;
+import com.intellij.refactoring.ui.MemberSelectionTable;
+import com.intellij.refactoring.util.classMembers.MemberInfo;
+import com.intellij.ui.ScrollPaneFactory;
+
+/**
+ *
+ * dialog for user to check fields mockable and methods testable
+ *
+ * @author huangliang
+ */
+public class CreateTestMeDialog extends DialogWrapper {
+
+    private final Map<String, Object> templateCtxtParams;
+    private final PsiClass myTargetClass;
+    private final String templateFileName;
+    private final MemberSelectionTable myMethodsTable = new MemberSelectionTable(Collections.emptyList(), null);
+    private final MemberSelectionTable myFieldsTable = new MemberSelectionTable(Collections.emptyList(), null);
+    private final List<String> checkedFieldNameList = new ArrayList<>();
+    private final List<String> checkedMethodIdList = new ArrayList<>();
+
+
+    public CreateTestMeDialog(@NotNull Project project, @NotNull @NlsContexts.DialogTitle String title,
+        PsiClass targetClass, String templateFileName, Map<String, Object> templateCtxtParams) {
+        super(project, true);
+        myTargetClass = targetClass;
+        this.templateCtxtParams = templateCtxtParams;
+        this.templateFileName = templateFileName;
+        setTitle(title);
+        init();
+    }
+
+    @Override
+    protected JComponent createCenterPanel() {
+        JPanel panel = new JPanel();
+        panel.setLayout(new BoxLayout(panel, BoxLayout.Y_AXIS));
+
+        JLabel fieldLabel = new JLabel("Select Need Mocked Fields");
+        panel.add(fieldLabel);
```

Comment:
would be good to avoid adding this label and panel in case class has no field that can be mocked